client/app/widgets/QuickCreate/index.jsx

```diff
@@ -35,7 +35,7 @@ const alpha = (a: string, b: string) => {
   return 0;
 };
 
-const sortAlpha = (checkboxes: Checkbox[]) =>
+const sortAlpha = (checkboxes: Checkbox[]): Checkbox[] =>
```

Comment:
Added this because Flow complains:
```
Missing type annotation for T.

     36│ };
     37│
     38│ const sortAlpha = (checkboxes: any[]) => (
     39│   checkboxes.sort((a: any, b: any) => alpha(a.label, b.label))
     40│ );
     41│
     42│ export class QuickCreate extends React.Component<Props, State> {

```

https://github.com/facebook/flow/issues/6151